packages/template-retail-react-app/package.json

```diff
@@ -88,6 +88,11 @@
   "devDependencies": {
     "cross-env": "^5.2.1"
   },
+  "overrides": {
```

Comment:
any reason why we are setting this overrides? This may be consider a hack to get around dependencies

Reply:
The alternative to this override is to update react-router. But within react-router, the dependency update occurred as part of a major version bump. And I don't think we want to be applying a major version bump of react-router at this time.